app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -1936,14 +1938,16 @@ object TelemetryWrapper {
             extras = [
                 TelemetryExtra(name = Extra.DELAY, value = "minutes"),
                 TelemetryExtra(name = Extra.MESSAGE_ID, value = "message_id"),
-                TelemetryExtra(name = Extra.LINK, value = "url|deeplink|null")
+                TelemetryExtra(name = Extra.LINK, value = "url|deeplink|null"),
+                TelemetryExtra(name = Extra.BACKGROUND, value = "true|false")
```

Comment:
As @cnevinc mentioned, we should use `,` to separate value instead of `|` in the document.